kratos/geometries/quadrilateral_2d_4.h

```diff
@@ -447,25 +445,36 @@ template<class TPointType> class Quadrilateral2D4
      */
     virtual double DomainSize() const override
     {
-        return fabs( this->DeterminantOfJacobian( PointType() ) ) * 0.5;
+        return std::abs( this->DeterminantOfJacobian( PointType() ) ) * 0.5;
     }
 
     /**
-     * Returns whether given arbitrary point is inside the Geometry
+     * Returns whether given arbitrary point is inside the Geometry and the respective 
+     * local point for the given global point
+     * @param rPoint: The point to be checked if is inside o note in global coordinates
+     * @param rResult: The local coordinates of the point
+     * @param Tolerance: The  tolerance that will be considered to check if the point is inside or not
+     * @return True if the point is inside, false otherwise
      */
-    virtual bool IsInside( const CoordinatesArrayType& rPoint, CoordinatesArrayType& rResult, const double Tolerance = std::numeric_limits<double>::epsilon() ) override
+    virtual bool IsInside( 
+        const CoordinatesArrayType& rPoint, 
+        CoordinatesArrayType& rResult, 
+        const double Tolerance = std::numeric_limits<double>::epsilon() 
+        ) override
     {
         this->PointLocalCoordinates( rResult, rPoint );
 
-        if ( fabs(rResult[0]) <= (1.0+Tolerance) )
-            if ( fabs(rResult[1]) <= (1.0+Tolerance) )
+        if ( std::abs(rResult[0]) <= (1.0+Tolerance) )
```

Comment:
Ok, we can restore the fabs, but geometry.h includes #include "utilities/math_utils.h", which includes cmath, which means that all geometries have the cmath included. We can continue discussion, but I decided to change it because I read in StackOverFlow that it is not a good practive to mix them